packages/darklang/prettyPrinter/programTypes.dark

```diff
@@ -883,7 +883,7 @@ module Darklang =
           | [] -> ""
           | _ ->
             p.declaration.typeParams
-            |> Stdlib.List.map (fun typeParam -> $"'{typeParam}")
+            |> Stdlib.List.map (fun typeParam -> $"{typeParam}")
```

Comment:
Hm - let's update WrittenTypes to record the `'` as a symbol, separate from the actual type variable.
Then, the pretty-printer probably needs the `'` to be included like the source was.